src/components/buttonMore/buttonmore.scss

```diff
@@ -0,0 +1,14 @@
+.button-more {
```

Comment:
please be consistent in naming files -- we use kebab-case for file naming -- so it should be button-more/button-more.scss